jukebox/src/main/webapp/java/com/spsteam14/jukebox/HelloAppEngine.java

```diff
@@ -0,0 +1,45 @@
+package com.spsteam14.jukebox;
+
+import com.google.appengine.api.utils.SystemProperty;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.Firestore;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.cloud.firestore.WriteResult;
+
+@WebServlet(value = "/hello")
+public class HelloAppEngine extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
is it possible to use a java formatter in the cloud environment? I'm seeing some inconsistencies and it would be good to run a command to get them all at once

recommendation: https://github.com/google/google-java-format

Reply:
Oops I accidentally included that. I will delete the file